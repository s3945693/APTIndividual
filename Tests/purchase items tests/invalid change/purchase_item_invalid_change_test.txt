1
6
2
i0001
500
6
9