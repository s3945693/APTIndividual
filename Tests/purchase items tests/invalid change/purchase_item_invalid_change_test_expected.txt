
Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Items Menu
----------
ID   |Name                                    | Available | Price  
-------------------------------------------------------------------
I0002|Apple Pie                               |20         |$ 3.00
I0003|Lemon Cheesecake                        |10         |$ 4.00
I0004|Lemon Meringue Pie                      |20         |$ 3.00
I0005|Lemon Tart                              |12         |$ 3.75
I0001|Meat Pie                                |50         |$ 3.50

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Coins Summary
-------------
Denomination    |    Count
---------------------------
5 Cents                  0
10 Cents                 0
20 Cents                 0
50 Cents                 0
1 Dollars                0
2 Dollars                16
5 Dollars                11
10 Dollars               5

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Purchase Item
-------------
Please enter the id of the item you wish to purchase: You have selected "Meat Pie - Yummy Beef in Gravy surrounded by pastry". This will cost you $ 3.50.
Please hand over the money - type in the value of each note/coin in cents.
Please enter or ctrl-d on a new line to cancel this purchase:
You still need to give us $3.50: Sorry, not enough change. Transaction reverted.

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): Coins Summary
-------------
Denomination    |    Count
---------------------------
5 Cents                  0
10 Cents                 0
20 Cents                 0
50 Cents                 0
1 Dollars                0
2 Dollars                16
5 Dollars                11
10 Dollars               5

Main Menu
    1. Display Items
    2. Purchase Items
    3. Save and Exit
Administrator-Only Menu:
    4. Add Item
    5. Remove Item
    6. Display Coins
    7. Reset Stock
    8. Reset Coins
    9. Abort Program

Select your option (1-9): 
Goodbye!

